README.md

```diff
@@ -0,0 +1,49 @@
+## Usage
+### Neovim
+This grammar can be setup within [Neovim](https://github.com/neovim/neovim/) to provide highlighting for the `.inp` files used in [TOPAS-Academic](http://www.topas-academic.net/). 
+It requires:
+- **Neovim 0.10** or newer.
+- Git
+- A C compiler
+1. Install [Nvim-treesitter](https://github.com/nvim-treesitter/nvim-treesitter) using your preferred package manager. With [vim-plug](https://github.com/junegunn/vim-plug), for example, add this command to your `init.vim`:
+      ```vimscript
+      Plug 'nvim-treesitter/nvim-treesitter', {'do': ':TSUpdate'}
+      ```
+    then restart Neovim and run `:PlugInstall`
```

Comment:
Think we can get rid of the description on how to install `nvim-treesitter/nvim-treesitter` and instead just link to their README / docs